tests/ssl-opt.sh

```diff
@@ -8907,6 +8907,23 @@ run_test    "TLS 1.3 m->G AES_128_GCM_SHA256      , RSA_PSS_RSAE_SHA256" \
             -c "mbedtls_ssl_tls13_process_certificate_verify() returned 0" \
             -c "HTTP/1.0 200 OK"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
```

Comment:
From the point of view of our TLS 1.3 client it is the same test as the one before thus I think we can live without it.